controllers/user.js

```diff
@@ -7,7 +7,70 @@ module.exports = {
       return res.status(200).send({ users });
     } catch (err) {
       console.log(err);
-      return res.status(err.status || 500).message(err.message || "Something went wrong...");
+      return res
+        .status(err.status || 500)
+        .message(err.message || "Something went wrong!");
     }
-  }
-};
\ No newline at end of file
+  },
+  getOne: async (req, res) => {
+    try {
+      const { user } = req;
+      return res.status(200).send({ user });
+    } catch (err) {
+      console.log(err);
+      return res
+        .status(err.status || 500)
+        .message(err.message || "Something went wrong!");
+    }
+  },
+  create: async (req, res) => {
+    try {
+      const { name, email } = req.body;
+
+      if (!name || !email) {
+        throw { status: 400, message: "Required fields can't be empty" };
+      }
+
+      const createdUser = await Users.create({
+        name,
+        email,
+      });
+
+      return res.status(200).send({ user: createdUser });
+    } catch (err) {
+      console.log(err);
+      return res
+        .status(err.status || 500)
+        .message(err.message || "Something went wrong!");
+    }
+  },
+  update: async (req, res) => {
+    try {
+      const { user, body } = req;
+      const { name } = body;
+
+      if (!name) {
+        throw { status: 400, message: "Required fields can't be empty" };
+      }
+
+      await Users.update(
+        {
+          name,
+        },
+        {
+          where: {
+            id: user.id,
+          },
+          individualHooks: true,
+        }
+      );
```

Comment:
@umairian we know that user in the req object is already a sequelize instance so we don't need to update the user by Model rather we can update the instance directly e.g
await req.user.update(payload,options)